paddle/fluid/framework/ir/fc_gru_fuse_pass.cc

```diff
@@ -48,7 +48,8 @@ static int BuildFusion(Graph* graph, const std::string& name_scope,
 
   // Create New OpDesc
   auto gru_creater = [&](Node* gru, Node* x, Node* weight_x, Node* weight_h,
-                         Node* bias, Node* hidden, Node* fc_bias) {
+                         Node* bias, Node* hidden, Node* fc_bias,
+                         const bool& use_mkldnn) {
```

Comment:
Maybe passing use_mkldnn as value would be better? Same with lstm_creater

Reply:
You are right, for such small types, it is better to pass as value.